common/app/common/NewNavigation.scala

```diff
@@ -379,16 +379,14 @@ object NewNavigation {
       }
     }
 
-    def getTopLevelSection(id: String) = {
-      val sectionList = sectionLinks.filter { item =>
+    def getParentPillar(id: String) = {
+      val sectionList = sectionLinks.find { item =>
         item.pageId == id
       }
 
-      if (sectionList.isEmpty) {
-        "News"
-      } else {
-        sectionList.head.parentSection.name
-      }
+      sectionList.map {
+        _.parentSection.name
+      }.getOrElse("News")
```

Comment:
It is only a matter of preference but I like to have one piece of logic per line. Something like:
```
sectionList
    .map(_.parentSection.name)
    .getOrElse("News")
```

You can even chain everything together:
```
sectionLinks
    .find(_.pageId == id)
    .map(_.parentSection.name)
    .getOrElse("News")
```